### ISSUE TITLE
Defining new one-letter symbols for unnatural amino acids

### ISSUE THREAD

**Issue by cxdummies** [cox-labs/CoxLab_Bug_Reporting#12]
Dear MaxQuant team,

Would it be possible to define new one-letter symbols for unnatural amino acids in MaxQuant? That would be very helpful.

**Comment by Arthfael** [comment]
I second that, this would be a great addition.

**Comment by yeroslaviz** [comment]
Not really. unnatural amino acids (UAAs) do not have standardized one-letter codes like the 20 standard amino acids. The one-letter code system is reserved for the canonical amino acids, which are encoded directly by the genetic code (e.g., A for Alanine, R for Arginine, etc.).

For UAAs, because they are not encoded in the genetic code and are not universally recognized across organisms, they do not have a universal shorthand. Instead, UAAs are typically referred to by their full chemical names or their abbreviations (e.g., "AzF" for p-azido-L-phenylalanine), or specific labels defined within a particular study or context.

Do you have an example as to how this should be done?
